gdk/PixbufLoader.custom

```diff
@@ -47,14 +49,18 @@
 			return this.Write (bytes, (ulong) count);
 		}
 
-		private void LoadFromStream (System.IO.Stream input)
-		{
-			byte [] buffer = new byte [8192];
-			int n;
+        const int bufSize = 8192;
+        static ByteArrayPool pool = new ByteArrayPool ();
```

Comment:
Sure, that wouldn't be a problem. We can add the API and then dep on it once it propagates :)